mne/io/brainvision/brainvision.py

```diff
@@ -836,3 +836,15 @@ def read_raw_brainvision(vhdr_fname, montage=None,
     return RawBrainVision(vhdr_fname=vhdr_fname, montage=montage, eog=eog,
                           misc=misc, scale=scale, preload=preload,
                           stim_channel=stim_channel, verbose=verbose)
+
+
+def _bv_parser(description, add_for_r=1000):
```

Comment:
can you add a dedicated test for this function? just to make explicit what type of description you want to handle.